applications/luci-app-mwan3/luasrc/model/cbi/mwan/globalsconfig.lua

```diff
@@ -3,61 +3,42 @@
 
 local net = require "luci.model.network".init()
 
-local s, m, mask, rtmon, rtlookup, logging, loglevel
+local s, m, o
 
 m = Map("mwan3", translate("MWAN - Globals"))
 
 s = m:section(NamedSection, "globals", "globals", nil)
 
-mask = s:option(
-	Value,
-	"mmx_mask",
+o = s:option(Value, "mmx_mask",
 	translate("Firewall mask"),
 	translate("Enter value in hex, starting with <code>0x</code>"))
-mask.datatype = "hex(4)"
-mask.default = "0xff00"
+o.datatype = "hex(4)"
+o.default = "0xff00"
 
-logging = s:option(Flag,
-	"logging",
+o = s:option(Flag, "logging",
 	translate("Logging"),
 	translate("Enables global firewall logging"))
 
-loglevel = s:option(
-	ListValue,
-	"loglevel",
+o = s:option(ListValue, "loglevel",
 	translate("Loglevel"),
 	translate("Firewall loglevel"))
-loglevel.default = "notice"
-loglevel:value("emerg", translate("Emergency"))
-loglevel:value("alert", translate("Alert"))
-loglevel:value("crit", translate("Critical"))
-loglevel:value("error", translate("Error"))
-loglevel:value("warning", translate("Warning"))
-loglevel:value("notice", translate("Notice"))
-loglevel:value("info", translate("Info"))
-loglevel:value("debug", translate("Debug"))
-loglevel:depends("logging", "1")
-
-rtmon = s:option(
-	Value,
-	"rtmon_interval",
-	translate("Update interval"),
-	translate("How often should rtmon update the interface routing table"))
-rtmon.datatype = "integer"
-rtmon.default = "5"
-rtmon:value("1", translatef("%d second", 1))
-rtmon:value("3", translatef("%d seconds", 3))
-rtmon:value("5", translatef("%d seconds", 5))
-rtmon:value("7", translatef("%d seconds", 7))
-rtmon:value("10", translatef("%d seconds", 10))
-
-rtlookup = s:option(DynamicList,
-	"rt_table_lookup",
+o.default = "notice"
+o:value("emerg", translate("Emergency"))
+o:value("alert", translate("Alert"))
+o:value("crit", translate("Critical"))
+o:value("error", translate("Error"))
+o:value("warning", translate("Warning"))
+o:value("notice", translate("Notice"))
+o:value("info", translate("Info"))
+o:value("debug", translate("Debug"))
+o:depends("logging", "1")
+
+o = s:option(DynamicList, "rt_table_lookup",
 	translate("Routing table lookup"),
 	translate("Also scan this Routing table for connected networks"))
-rtlookup.datatype = "integer"
-rtlookup:value("1", translatef("Routing table %d", 1))
-rtlookup:value("2", translatef("Routing table %d", 2))
-rtlookup:value("220", translatef("Routing table %d", 220))
+o.datatype = "integer"
+o:value("1", translatef("Routing table %d", 1))
+o:value("2", translatef("Routing table %d", 2))
```

Comment:
I know that this was broken before your fixes, but "1" and "2" should not be default options here since those routing tables are very likely going to be used by mwan3

Reply:
The main issue here was to enter the routes for IPsec in table 220. And mwan3 should have a look at this too. This was a workaround with IPsec. The question is, of course, do we still need it? For that i have to look at IPsec with all the changes in mwan3 again we have done recently. I would leave it like this for now

Reply:
I'm not sure if this feature is still needed. If it is, table "220" seems fine to leave as a default. However, there is no reason for a user to add routing table 1 or 2 here, since tables 1 and 2 are very likely going to be used by mwan3 for the first and second interfaces.